cmd/mpi-operator.v1/app/options/options.go

```diff
@@ -67,4 +68,6 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&s.GangSchedulingName, "gang-scheduling", "", "Set gang scheduler name if enable gang scheduling.")
 
 	fs.StringVar(&s.LockNamespace, "lock-namespace", "mpi-operator", "Set locked namespace name while enabling leader election.")
+
+	fs.BoolVar(&s.LauncherRunWorkload, "launcher-run-workload", false, "Set launcher run the workload when launcher has GPU")
```

Comment:
LauncherRunsWorkloads (with plurals). Are there better names for this?

Reply:
launcher-with-gpu ? I have no idea about it.

Reply:
done